### ISSUE TITLE
Error in readme documentation 

### ISSUE THREAD

**Issue by v-flashpoint** [livekit/client-sdk-js#870]
### Describe the bug

Hi all,

Just wanted to flag up an error in the initial documentation on the front page.

You have 

```
function handleLocalTrackUnpublished(track: LocalTrackPublication, participant: LocalParticipant) {
  // when local tracks are ended, update UI to remove them from rendering
  track.detach();
}
```

But that throws an error, as track.detach() is not a function. It looks like what you need to do is something like this

```
function handleLocalTrackUnpublished(localtrack: LocalTrackPublication, participant: LocalParticipant) {
  // when local tracks are ended, update UI to remove them from rendering
  localtrack.track.detach();
}
```

Hope that helps!

### Reproduction

Open two tabs using the same access token. One will cause the other to disconnect, triggering the error.

### System Info

```shell
Windows, WAMP
```

### Severity

annoyance
